peps/pep-0506.rst

```diff
@@ -213,7 +213,7 @@ module [#]_.  This received considerable scepticism and outright opposition:
   which a CSPRNG lacks by design.
 
 * Another major use of the ``random`` module is for simple "guess a number"
-  games written by beginners, and many people are loath to make any
```

Comment:
This one's already correct, the edit makes it incorrect.

Reply:
Thanks, great catch! That's a bug in codespell, I'll report and fix there